packages/extension-task-item/src/task-item.ts

```diff
@@ -67,7 +67,7 @@ export const TaskItem = Node.create<TaskItemOptions>({
         keepOnSplit: false,
         parseHTML: element => element.getAttribute('data-checked') === 'true',
```

Comment:
I wonder if it is the `parseHTML` that should be more lenient instead I don't think that it needs to be checked against true, just that it actually exists right?

Reply:
Yea so my testing shows that when data-checked is on the element (i.e. generateHTML outputted `<li data-checked>...`) that it gives an empty string to say that it was set but has no vlaue.

As opposed to being completely empty which is just null.

So we should be more liberal in our parsing rather than more strict in our output and the result should be the same:

<img width="250" alt="image" src="https://github.com/user-attachments/assets/4f3865f8-c98e-4e72-8694-d79334f20c48">

Reply:
Please see my REPL repo. https://github.com/baseballyama/tiptap-tasklist-bug-repl
And actually there is `data-checked='false'` attribute.

![image](https://github.com/user-attachments/assets/1bd4e467-3022-4447-99b2-7fb3e99fec71)